spiffetls/tests/check_listen.c

```diff
@@ -0,0 +1,78 @@
+#include "spiffetls/src/listen.h"
+#include <check.h>
+#include <unistd.h>
+
+void *call_client(void *unused)
+{
+    sleep(1);
+    system("./tls_client 20001 &");
+    pthread_exit(NULL);
+    return NULL;
+}
+
+START_TEST(test_spiffetls_ListenWithMode)
```

Comment:
What do you think about commenting the test details like precondition and post condition?